cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -278,8 +281,8 @@ public void execute() {
                 // compile the modules
                 .addTask(new CompileTask(outStream, errStream, false, isPackageModified, buildOptions.enableCache()))
 //                .addTask(new CopyResourcesTask(), listGroups) // merged with CreateJarTask
-                .addTask(new RunTestsTask(outStream, errStream, rerunTests, groupList, disableGroupList,
-                        testList, includes, coverageFormat, moduleMap, listGroups))
+                .addTask(new RunTestsTask(outStream, errStream, rerunTests, groupList, disableGroupList, testList,
+                        includes, coverageFormat, moduleMap, listGroups, project.buildOptions().nativeImage()))
```

Comment:
We can read the buildoption from the project in `RunTestsTask.execute` method. We dont need to pass it as an argument to the constructor.

Reply:
Reading the build option rather than passing it directly